actions/shared_publish_setup/action.yaml

```diff
@@ -8,8 +8,13 @@ inputs:
   java_version:
     description: The version of Java to use to compile the JAR. Defaults to 11
   version_number_input:
-    description: If set, the version number will not be incremented and the given number will be used.
```

Comment:
The description was wrong - this value had v prepended and then that was used as the git ref to checkout. It wouldn't work to check out a specific hash or anything else. As far as I can find, it was never used anywhere.